### ISSUE TITLE
Android 13

### ISSUE THREAD

**Issue by lulateamo** [ZaneYork/SMAPI-Android-Installer#315]
not wanting to be annoying, but will the next smapi update be back to normal?  because I use Android 13 and impossible to access the data folder.

**Comment by Rubb-le** [comment]
? I have Android 13 too, I accessed the data folder using this file manager https://play.google.com/store/apps/details?id=com.marc.files&pli=1

**Comment by Saucison78** [comment]
You use fv file manager and it works fine, however you have to use the game listed in the guide in order to get the com.zane.stardewvalley folder listed, the official game will no longer work thanks to google's new policy.

**Comment by lulateamo** [comment]
> ? Eu também tenho o Android 13, acessei a pasta de dados usando este gerenciador de arquivos https://play.google.com/store/apps/details?id=com.marc.files&pli=1

yes, I managed to access the data folder, but I can't extract the mods, because the manager asks to extract with another file manager and I'm not able to extract the mods.  it's kinda confusing

**Comment by lulateamo** [comment]
> Você usa o gerenciador de arquivos fv e funciona bem, no entanto, você deve usar o jogo listado no guia para obter a pasta com.zane.stardewvalley listada, o jogo oficial não funcionará mais graças à nova política do Google.

i hate google

**Comment by Ryanhwr** [comment]
Psr, mas eu creio q quando lançar o smapi 100%, N vamos mais usar aquela pasta "data" cujo o Android 13 fez questão de restritar, acho q poderemos mover os arquivos pra outro lugar exceto no "data".

**Comment by NikovLaguthy** [comment]
Hello guys. I'm here to give one recommendation to everyone with Android 13! The app "Zarchiver" is perfect to acess and edit the content on android session. Its working perfectly nowadays, i use it and i recommend!

**Comment by DollSkies** [comment]
> Hello guys. I'm here to give one recommendation to everyone with Android 13! The app "Zarchiver" is perfect to acess and edit the content on android session. Its working perfectly nowadays, i use it and i recommend!

I have tried every file manager that people have recommended on this, and I still can acess my data. I need help (I have an A72 if that makes a difference)

**Comment by DollSkies** [comment]
> You use fv file manager and it works fine, however you have to use the game listed in the guide in order to get the com.zane.stardewvalley folder listed, the official game will no longer work thanks to google's new policy.

What game listed in the guide? I've looked three times and I only see com.zane and the smapi apk

**Comment by NikovLaguthy** [comment]
> > Olá, pessoal. Estou aqui para dar uma recomendação a todos com Android 13! O aplicativo "Zarchiver" é perfeito para acessar e editar o conteúdo na sessão android. Está funcionando perfeitamente hoje em dia, eu uso e recomendo!
> 
> Eu tentei todos os gerenciadores de arquivos que as pessoas recomendaram sobre isso e ainda posso acessar meus dados. Preciso de ajuda (eu tenho um A72 se isso faz diferença)

in that case, you can try "Mt Manager". It can move contents, work exactly for android 13, your use and interface its a litle complicated, But you just need read with patience and you will can edit your contents with no problem. if you found some trouble, try to contact me on twt @@souTevan
